### ISSUE TITLE
Infinite loop in vertical_whitespace

### ISSUE THREAD

**Issue by garrett-davidson** [realm/SwiftLint#817]
When opting in to vertical_whitespace checking, the following code cause SwiftLint to seemingly get stuck in an infinite loop and never exit.

```
// Some random comment with strange formatting /**                                                                                                 
public func useMysql() {
    guard dataMysql.query(statement: "\(createUsersTableQuery)") else {
        print("Error creating users table")
        return
    }

    guard dataMysql.query(statement: "\(createMessagesTableQuery)") else {
        print("Error creating messages table")
        return
    }

    guard dataMysql.query(statement: "\(createFacebookTableQuery)") else {
        print("Error creating facebook table")
        return
    }

    guard dataMysql.query(statement: "\(createGoogleTableQuery)") else {
        print("Error creating google table")
        return
    }
}
```

I was able to narrow this chunk of code down from a much larger file. It seems like removing any of the remaining lines causes the problem to no longer reproduce.

**Comment by norio-nomura** [comment]
Have this already been fixed by #775?

**Comment by masters3d** [comment]
@garrett-davidson Which version of Swiftlint are you using? 
`// Some random comment with strange formatting /**                                                                                                 
`
there was a regex multiline comment bug in an older version but that has been fixed.

**Comment by garrett-davidson** [comment]
This happens in 0.12.0, the latest GitHub release.

**Comment by masters3d** [comment]
I am not not seeing a bug. Could you share the commented out section? 
```
Linting Swift files in current working directory
Linting 'possiblebug.swift' (1/1)
possiblebug.swift:1: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
possiblebug.swift:2: warning: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
Done linting! Found 2 violations, 0 serious in 1 file.
```

**Comment by garrett-davidson** [comment]
[test.zip](https://github.com/realm/SwiftLint/files/548844/test.zip)

I just reproduced it with this exact setup.

**Comment by masters3d** [comment]
Looks to me like that bug was fixed on master but it hasn't gotten pushed as a release. 
a work around is to remove `/**` from the end of the comment.

``// Some random comment with strange formatting ``

**Comment by jpsim** [comment]
This was fixed in 0.13.0.
